src/cargo-about/generate.rs

```diff
@@ -212,6 +212,7 @@ fn generate(
             lic.used_by.sort_by(|a, b| a.krate.id.cmp(&b.krate.id));
         }
 
+        licenses.sort_by(|a, b| a.used_by[0].krate.name.cmp(&b.used_by[0].krate.name));
```

Comment:
I think here we would want to do the same ordering as directly above using the full ID, since that includes the version as well.

Reply:
Good call, updated